frontend/src/test/scala/bloop/tasks/ProjectHelpers.scala

```diff
@@ -216,7 +217,8 @@ object ProjectHelpers {
         }
 
         override def postVisitDirectory(dir: Path, exc: IOException): FileVisitResult = {
-          Files.delete(dir)
+          try Files.delete(dir)
+          catch { case _: DirectoryNotEmptyException => () } // Happens sometimes on Windows?
```

Comment:
This makes me uncomfortable... I don't want bloop become sbt in being super defensive on IO because of Windows support. Do you know exactly why this happens and in which scenario?

Reply:
It happens in the file watching tests, but I don't know anything more...

Reply:
I'll look a bit further tomorrow, but I don't really know what to do about that. I strongly suspect that Windows thinks that the directory is not empty because of the file watching, but I can't tell for sure.

Note that this is only in test code, when the test is done and everything has run smoothly.